File: app/src/main/java/basic/ex01.java
package basic;

public class ex01 {
    public static void main(String[] args) {
        System.out.println("Hello, Java World!");

        for (int i = 0; i < args.length; i++) {
            System.out.println("args[" + i + "]: " + args[i]);
        }
    }
}

File: app-kotlin/src/main/kotlin/basic/ex01.kt
package basic

class ex01 {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            println("Hello, Java World! (Kotlin)")
            args.forEachIndexed { index, value ->
                println("args[$index]: $value")
            }
        }
    }
}
